### PR TITLE
Add documentation for Neptune Apex

### PR DESCRIPTION
**Pull request by jpelzer**
This reverts commit e723871844ee3fc6296cf9711f5ed0daf44930e3.

## Proposed change

Adds documentation for new integration Neptune Apex, the Neptune System Apex Aquacontroller, an aquarium automation system. 

This documentation was prematurely merged and reverted prior to the integration being accepted into core (https://github.com/home-assistant/home-assistant.io/pull/13138), so this pull request is a revert of the reversion, and should only be merged when the parent pull has been accepted.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/34558
- Link to parent pull request in the Brands repository: PR needs re-reversion, will update

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/neptune_apex.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  type: string
+username:
+  description: The username required to connect to the Apex.
+  required: true
+  type: string
+password:
+  description: The password required to connect to the Apex.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (frenck) [inline source/_integrations/neptune_apex.markdown]: This configuration block is used for YAML configuration. Please remove it.

File: source/_integrations/neptune_apex.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Sensor
+
+Once you have enabled the **Neptune Apex** component, your available Apex probes will automatically populate and be refreshed every 30 seconds. Apex probes can report data about temperature, pH, ORP, amperage, flow rate, water detection, and many others. Rather than create custom types for every possible data option, this integration reports the raw probe type as reported by the Apex as the sensor's `unit_of_measurement`. If the probe type is reported as 'Temp', the sensor's `device_class` will be set to 'temperature'. If the probe type is reported as 'Amps', the sensor's `device_class` will be set to 'power'.
+
+## Light
```
Comment (frenck) [inline source/_integrations/neptune_apex.markdown:37]:
```suggestion
## Light

```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @jpelzer!

Reviewed your PR and left 2 small suggestions. Could you take a look? Thanks! 👍

File: source/_integrations/neptune_apex.markdown (hunk 1 above)
Comment (jpelzer) [reply source/_integrations/neptune_apex.markdown]: Done.

File: source/_integrations/neptune_apex.markdown (hunk 2 above)
Comment (jpelzer) [reply source/_integrations/neptune_apex.markdown:37]: Accepted.

File: source/_integrations/neptune_apex.markdown [hunk 3]
```diff
@@ -24,26 +24,17 @@ There is currently support for the following device types within Home Assistant:
...
-username:
-  description: The username required to connect to the Apex.
-  required: true
-  type: string
-password:
-  description: The password required to connect to the Apex.
-  required: true
-  type: string
-{% endconfiguration %}
+In the popup:
```
Comment (klaasnicolaas) [inline source/_integrations/neptune_apex.markdown:27]:
```suggestion
In the popup:

```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/neptune_apex.markdown (hunk 3 above)
Comment (jpelzer) [reply source/_integrations/neptune_apex.markdown:27]: Updated.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**